cmd/protoc-gen-go-http/httpTemplate.tpl

```diff
@@ -25,18 +25,11 @@ func Register{{.ServiceType}}HTTPServer(s *http.Server, srv {{.ServiceType}}HTTP
 func _{{$svrType}}_{{.Name}}{{.Num}}_HTTP_Handler(srv {{$svrType}}HTTPServer) func(ctx http.Context) error {
 	return func(ctx http.Context) error {
 		var in {{.Request}}
-		{{- if .HasBody}}
-		if err := ctx.Bind(&in{{.Body}}); err != nil {
-			return err
-		}
-
-		{{- if not (eq .Body "")}}
-		if err := ctx.BindQuery(&in); err != nil {
+		if err := ctx.BindQuery(&in{{.Body}}); err != nil {
```

Comment:
Should use `ctx.BindQuery(&in)`, At this priority:
```
ctx.Bind(&in{{.Body}})
ctx.BindQuery(&in)
ctx.BindVars(&in)
```

Reply:
Okay, I've fixed it